idp-binding-service/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## ------------------------------------------ IdP (oauth & openid) config ----------------------------------------------
+
+## supported scopes
+mosip.idp.supported.authorize.scopes={'resident-service'}

```

Comment:
Take the latest updated these properties are removed.

Reply:
Removed unused properties from this file.